logic/strategy/src/lib.rs

```diff
@@ -40,6 +40,7 @@
 //! ```
 
 use hopr_primitive_types::prelude::*;
+use one_hop_connection::OneHopConnectionStrategyConfig;
```

Comment:
Maybe a better name would be `path_optimizing` strategy, since it's aim should be to make sure that the channel graph paths are optimized in a way to allow optimal anonymous path contruction to target peers.